docs/excel/custom-functions-save-state.md

```diff
@@ -36,15 +40,60 @@ function GetValue(key) {
 
 [A more detailed code sample on GitHub](https://github.com/OfficeDev/PnP-OfficeAddins/tree/master/Excel-custom-functions/AsyncStorage) gives an example of passing this information to the task pane.
 
->[!NOTE]
-> The `storage` object replaces the previous storage object named `AsyncStorage` which is now deprecated. If using the `AsyncStorage` object in your current custom functions code, please update it to use the `storage` object.
+The following methods are available on the `storage` object:
+
+- getItem
+- getItems
+- setItem
+- setItems
+- removeItem
+- removeItems
+- getKeys
+
+> [!NOTE]
+There's no method for clearing all information (such as `clear`). Instead, use `removeItems` to remove multiple entries at a time.
+
+> [!NOTE]
+> The JavaScript-only runtime doesn't have access to the Document Object Model (DOM), so it can't use jQuery or other libraries which rely on the DOM.
+
+## Requesting external data
+
+Within a custom function, you can request external data by using an API like [Fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) or by using [XmlHttpRequest (XHR)](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest), a standard web API that issues HTTP requests to interact with servers.
+
+Within the JavaScript runtime used by custom functions, XHR implements additional security measures by requiring [Same Origin Policy](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy) and simple [CORS](https://www.w3.org/TR/cors/).
+
+Note that a simple CORS implementation cannot use cookies and only supports simple methods (GET, HEAD, POST). Simple CORS accepts simple headers with field names `Accept`, `Accept-Language`, `Content-Language`. You can also use a `Content-Type` header in simple CORS, provided that the content type is `application/x-www-form-urlencoded`, `text/plain`, or `multipart/form-data`.
+
+## Finding an addressing cell's context parameter
```

Comment:
Also, the term "addressing" seems weird. I'm not exactly sure what it means. It that used a lot in this doc set?